Formula/sshcode.rb

```diff
@@ -0,0 +1,28 @@
+class Sshcode < Formula
+  desc "Run VS Code on any server over SSH"
+  homepage "https://github.com/cdr/sshcode"
+  url "https://github.com/cdr/sshcode/archive/v0.8.0.tar.gz"
+  sha256 "235dc537d8b2d1425a8c90fc9ccc1f404cd01a03c6c04f2e7f1a0b48521e59aa"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOOS"] = "darwin"
+    ENV["GOARCH"] = "amd64"
+    ENV["GOPATH"] = buildpath
+    ENV["GOBIN"] = bin
+    ENV["GO111MODULE"] = "on"
+    sshcodepath = buildpath/"src/github.com/cdr/sshcode"
+    sshcodepath.install buildpath.children
+
+    cd "src/github.com/cdr/sshcode" do
+      system "go", "build", "-o", "build/sshcode", "."
+      bin.install "build/sshcode"
+      prefix.install_metafiles
+    end
+  end
+
+  test do
+    system "#{bin}/sshcode", "--version"
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (`foo --version` or `foo --help`) are not sufficient, as explained in the [formula cookbook](https://docs.brew.sh/Formula-Cookbook).

In most cases, a good test would involve running a simple test case: run `#{bin}/foo input.txt`.
- Then you can check that the output is as expected (with `assert_equal` or `assert_match` on the output of `shell_output`)
- You can also check that an output file was created, if that is expected: `assert_predicate testpath/"output.txt", :exist?`

Some advice for specific cases:
- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.